pkg/operator/cluster/ceph/osd/osd.go

```diff
@@ -169,18 +182,20 @@ func (c *Cluster) Start() error {
 			continue
 		}
 
-		// create the replicaSet that will run the OSDs for this node
-		rs := c.makeReplicaSet(n.Name, n.Devices, n.Selection, resources, n.Config)
-		_, err := c.context.Clientset.Extensions().ReplicaSets(c.Namespace).Create(rs)
+		// create the pod that will run the OSDs for this node
+		logger.Infof("creating osd pod for node %s", n.Name)
+		pod := c.makePod(n.Name, n.Devices, n.Selection, resources, n.Config)
```

Comment:
Rather than a pod we need to create a deployment to manage the pods for us, right?